builder/transaction_builder.go

```diff
@@ -41,14 +42,17 @@ type TxInput struct {
 	// The ID of the referenced input.
 	InputID iotago.OutputID `json:"inputID"`
 	// The output which is used as an input.
-	Input iotago.Output `json:"input"`
+	Input iotago.OutputWithCreationTime `json:"input"`
```

Comment:
I think this needs to be reverted, the `OutputWithCreationTime` is something just the VM needs